src/adapters/github.js

```diff
@@ -102,17 +103,25 @@ class GitHub extends PjaxAdapter {
   }
 
   // @override
-  updateLayout(sidebarPinned, sidebarVisible, sidebarWidth) {
+  updateLayout(sidebarPinned, sidebarVisible, sidebarWidth, size = 'left') {
     const SPACING = 10;
     const $header = $(GH_HEADER);
     const $containers = $(GH_CONTAINERS);
     const autoMarginLeft = ($(document).width() - $containers.width()) / 2;
     const shouldPushEverything = sidebarPinned && sidebarVisible;
     const smallScreen = autoMarginLeft <= sidebarWidth + SPACING;
 
-    $('html').css('margin-left', shouldPushEverything && smallScreen ? sidebarWidth : '');
-    $containers.css('margin-left', shouldPushEverything && smallScreen ? SPACING : '');
-    $header.css('padding-left', shouldPushEverything && !smallScreen ? sidebarWidth : '');
+    $('html').removeAttr('style');
+    $containers.removeAttr('style');
+    $header.removeAttr('style');
+
+    $('html').css(`margin-${size}`, shouldPushEverything && smallScreen ? sidebarWidth : '');
+    $containers.css(`margin-${size}`, shouldPushEverything && smallScreen ? SPACING : '');
+
+    if (shouldPushEverything && !smallScreen) {
```

Comment:
You're sure this works fine with both large and small screens?

Reply:
The change wouldn’t affect small screens. There are minor changes including adding padding and ‘important’ for large screens.